clusterloader2/pkg/measurement/common/slos/network_programming.go

```diff
@@ -48,6 +48,10 @@ func init() {
 type netProgGatherer struct{}
 
 func (n *netProgGatherer) IsEnabled(config *measurement.MeasurementConfig) bool {
+	// Disable NetworkProgrammingLatency measurement if scraping kube-proxy is disabled.
+	if nil != config && !config.ClusterLoaderConfig.PrometheusConfig.ScrapeKubeProxy {
```

Comment:
NIT: While checking whether a variable isn't nil is a good practice in general, you can assume that it won't be nil here, so you can remove this check.